internal/pubsub/pubsub.go

```diff
@@ -206,8 +217,12 @@ func NewSubscription(d driver.Subscription) *Subscription {
 			id := m.ackID
 			ids = append(ids, id)
 		}
+		callCtx := context.TODO()
+		err := retry.Call(callCtx, gax.Backoff{}, d.IsRetryable, func() error {
+			return d.SendAcks(callCtx, ids)
+		})
 		// TODO(#695): Do something sensible if SendAcks returns an error.
```

Comment:
Remove the TODO?

Reply:
We still have to make a failed SendAcks affect Receive and Close.